database/seed.js

```diff
@@ -26,7 +27,7 @@ const createData = async function() {
           age: (Math.floor(Math.random() * 20)) + 30,
           what_you_liked: faker.lorem.words(),
           what_you_did_not_liked: faker.lorem.sentence(),
-          fit: Math.floor(Math.random() * 7),
+          fit: Math.floor(Math.random() * 7)
```

Comment:
Is your fit property a number? The original website has options like "relaxed", "tight", "waist length", etc.

Unless it's a rating thing, but I find it odd that it's a number between 0-7

Reply:
@matt-leyrat yes, it's a slide bar with 7 points and I'm planning to render them on the screen according to the number!

Reply:
Sweet! Thanks for clarifying